packages/core/test/actor.test.ts

```diff
@@ -487,7 +491,10 @@ describe('spawning observables', () => {
         idle: {
           entry: assign({
             observableRef: ({ spawn }) => {
-              const ref = spawn(observableLogic, { id: 'int' });
+              const ref = spawn(observableLogic, {
+                id: 'int',
+                syncSnapshot: true
```

Comment:
It's worth noting that we don't have any means right now to validate that this is actually safe on the type-level